app/scripts/modules/core/src/reactShims/react.uirouter.css

```diff
@@ -0,0 +1,4 @@
+react-ui-view-adapter {
```

Comment:
an extra custom element is created inside a `<ui-view>` which messes up the layout.  This fixes it (for load balancers, but we may need to tweak selectors going forward)